docs/_content/ontwikkelaars/algemeen/archiveren.md

```diff
@@ -49,15 +53,100 @@ gerelateerde entiteiten:
 
 ### Wat gebeurt er als een zaak-dossier *blijvend bewaard* wordt?
 
-Het zaak-dossier is overgedragen naar een bewaarplaats.
+Het zaak-dossier wordt overgedragen naar een bewaarplaats.
 
 ### Wat gebeurt er als een zaak-dossier *vernietigd* wordt?
 
 Het vernietigen is het definitief verwijderen van data volgens de NEN2082 (Eis 80):
 
 > Vernietigen van archiefstukken/archiefbestanddelen moet zo gebeuren dat deze niet meer op enigerlei wijze kunnen worden gereproduceerd.
 
-## Gerelateerde attributen
+### Bepalen van de aan archivering gerelateerde attributen voor `ResultaatType`
+
+Zie: [Selectielijst gemeenten en intergemeentelijke organen 2017](https://vng.nl/files/vng/20170706-selectielijst-gemeenten-intergemeentelijke-organen-2017.pdf)
+
+Het ZTC dient ingericht te worden volgens de selectielijst. Het ZTC dient validaties uit te voeren om te zorgen dat inrichting correct is.
+
+*Voorbeeld*
+
+Als `ResultaatType.BrondatumArchiefprocedure.Afleidingswijze` wordt ingesteld op `eigenschap` moet het ZTC valideren dat het betreffende `ZaakType` een `Eigenschap` heeft met als `naam`, de waarde die staat in `ResultaatType.BrondatumArchiefprocedure.datumkenmerk`.
+
+### Berekenen van de `Zaak.Archiefactiedatum`
+
+1. Bepaal de *brondatum* van de `Zaak` door de `ResultaatType.BrondatumArchiefprocedure.Afleidingswijze` te raadplegen:
+
+   Afleidingswijze | Waarde van *brondatum*
+   --- | ---
+   `afgehandeld` | `Zaak.Einddatum`
+   `gerelateerde zaak` | *TODO: Wat is dit precies?* De hoogste datum van van alle `Zaak.GerelateerdeZaak.Einddatum` of `Zaak.Einddatum` ([#776](https://github.com/VNG-Realisatie/gemma-zaken/issues/776))
```

Comment:
Ik denk de hoogste van de einddata van gerelateerde zaken.

Reply:
Ja maar, wat zijn gerelateerde zaken in RGBZ termen?

Reply:
Op zoek in het RGBZ kom ik de relatie `ZAAK heeft gerelateerde ZAAK` tegen in tekst op p. 7. 

> Soms heeft de ene zaak betrekking op een andere zaak, wat we modelleren met de relatie ‘ZAAK heeft gerelateerde ZAAK’. Denk bijvoorbeeld aan een bezwaar of beroep dat naar aanleiding van een beschikking wordt ingediend en dat als een separate zaak wordt afgehandeld. De aard van de relatie tussen zaken leggen we vast met ZAKENRELATIE.

Maar ik kom het niet tegen bij de attributen van ZAAK. Wel deze:
> ZAAK [ 0 .. * ] heeft relevante andere ZAAK [ 0 .. * ] 

Laten we een resource gerelateerde zaken maken en daarbij ZAKENRELATIE als enum gebruiken. RGBZ p. 574 geeft de waarden voor de aard van de relatie:

| vervolg | De andere zaak gaf aanleiding tot het starten   van de onderhanden zaak.
-- | --
onderwerp | De andere zaak is relevant voor cq. is   onderwerp van de onderhanden zaak.
bijdrage | Aan het bereiken van de uitkomst van de andere   zaak levert de onderhanden zaak een bijdrage.